### DISCUSSION TITLE
Share deep link markdown snippet is missing an opening square bracket

### DISCUSSION THREAD

**Discussion by jimbobbennett** [community/community#54276] [Codespaces]
### Select Topic Area

Bug

### Body

**Steps to reproduce**

* From a repo, select Code -> Codespace -> Share a deep link
* From the snippets section select the Markdown Tab
* Copy he markdown and use it on a repo

**Expected**

* The markdown for the snippet should be correct

**Actual**

* The markdown is missing the first opening square bracket

Check out the image below:
![image](https://user-images.githubusercontent.com/1710385/235739575-69d8135f-4f54-487f-b8d6-0459bab1612b.png)

The markdown created gives you this:

![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/jimbobbennett/jims-blog)

And it should be:

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/jimbobbennett/jims-blog)

**Comment by Fengyi-Li** [comment]
Thanks for reporting this. The Codespace team will work on fixing.

**Comment by abourget** [comment]
hey, btw I get an invalid cert for codespaces.new !
